TOOLS_CHOICES.md

```diff
@@ -0,0 +1,46 @@
+# Tool Choices
+The right tool choices are key to a good project.
+# CI/CD tool
+These build tools are evaluated:
+- Github Actions
+- Travis CI
+- Jenkins
+
+## Checks for determining CI/CD tool
+- Must be available to the community
+- No/less maintenance
+- Integratable with Github (because that's where our repositories are)
+
+## Github Actions
+### Pros
+- Upcoming tool, really active in developing
+- Free tool for public Github repositories (which CoMPAS is)
+- Integrates perfectly with Github repositories
+
+### Cons
+- Software is proprietary
+- Not much experience within Alliander
```

Comment:
Instead of Alliander exepriance, you can say adoption is growing

Reply:
Added!